docs/guides/internet-access.md

```diff
@@ -0,0 +1,59 @@
+[](){#ref-guides-internet-access}
+# Internet Access on Alps
+
+The [Alps network][ref-alps-hsn] is mostly configured with private IP addresses (`172.28.0.0/16`).
+Login nodes have public IP addresses which means that they can directly access the internet, while a proxy server provides internet access for compute nodes.
+
+??? info "Compute node proxy configuration"
+
+    Compute nodes are configured with the following environment variables to use the proxy server:
+    
+    ```bash
+    https_proxy=http://proxy.cscs.ch:8080
+    http_proxy=http://proxy.cscs.ch:8080
+    no_proxy=.local, .cscs.ch, localhost, 148.187.0.0/16, 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16
+    HTTPS_PROXY=http://proxy.cscs.ch:8080
+    HTTP_PROXY=http://proxy.cscs.ch:8080
+    NO_PROXY=.local, .cscs.ch, localhost, 148.187.0.0/16, 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16
+    ```
+
+!!! warning "Public IPs are a shared resource"
+    Be aware that public IPs, whether on login nodes or through the proxy, are essentially a shared resource.
+    Many services will rate limit or block usage based on the IP address if abused.
+    An example is pulling container images from Docker Hub.
+    [Authenticating with Docker Hub][ref-ce-third-party-private-registries] makes their rate limit apply per user instead.
+
+## Using SSH through the proxy server 
+
+While use of the proxy server is transparent for most use cases, others need additional configuration for compute nodes.
+An example is cloning git repositories from GitHub over SSH.
+Cloning over https works without additional configuration.
+To make SSH use the proxy server, add the following to your `~/.ssh/config` file:
+
+```bash
+Match Host *,!148.187.0.0/16,!192.168.0.0/16,!172.16.0.0/12,!10.0.0.0/8 exec "hostname -I | grep -vqF 148.187."
+    ProxyCommand nc -X connect -x proxy.cscs.ch:8080 %h %p
+```
+
+This configuration takes into account that login and compute nodes require a different setup.
+
+??? info "Error message when cloning without the proxy set up for SSH"
```

Comment:
`warning` instead of `info` will make it stand out more?

Reply:
My thinking was that it's just "info" as something to show up in search results. The whole section is a "warning" in a sense, saying "warning: to be able to clone on compute nodes you have to set up this config". That said, I'm happy with "warning" as well, really not a big deal for me.

Reply:
You are right that it isn't a big deal.

I thought that because the box contains an description of an error and what to do, having the orange title with an exclamation mark would make it stand out more on the page.

Reply:
That's fair! Made it a warning in https://github.com/eth-cscs/cscs-docs/pull/39/commits/fd29d208357a6e539cfc06e1df9591dcd0b2b6c6. Didn't mean to be difficult.